liquidity-rewards/contracts/UniswapPoolRewards.sol

```diff
@@ -56,6 +56,7 @@ contract UniswapPoolRewards is PoolTokenWrapper, AccessControl {
     }
 
     function setRewardRatePerSecond(uint256 rewardRatePerSecond_) public onlyRole(DEFAULT_ADMIN_ROLE) {
+        _updateReward();
         rewardRatePerSecond = rewardRatePerSecond_;
```

Comment:
@hhio618 Since `_updateReward()` uses the `rewardRatePerSecond` variable, shouldn't `_updateReward()` be called _after_ the adjustment?
```suggestion
        rewardRatePerSecond = rewardRatePerSecond_;
        _updateReward();
```

Reply:
In the scenario in #42 we wanted to preserve current user rewards after adjustment. We do it by saving the current rewardPerTokenDeposited value and the time when it happened (`lastUpdateTime`). If we first update the `rewardRatePerSecond` value, the current user's rewards will fluctuate.